test/puppetlabs/puppetdb/time_test.clj

```diff
@@ -1,4 +1,5 @@
 (ns puppetlabs.puppetdb.time-test
+  (:import (org.joda.time DateTime DateTimeZone))
```

Comment:
I think we usually have `:import` after the `:require`. Not sure if that has a reason or if we want to try to maintain consistency. Maybe @rlb will know?

Reply:
Yeah, imports normally go second.

Reply:
Okay, I'll move them in this file and time.clj.